components/table/demo/table.html

```diff
@@ -8,9 +8,6 @@
 			import '../../demo/demo-page.js';
 			import './table-test.js';
 		</script>
-		<script>
-			window.D2L = { LP: { Web: { UI: { Flags: { Flag: () => true } } } } }; // TODO: remove with GAUD-131-dropdown-fixed-positioning flag clean up
```

Comment:
This cause a bunch of confusion due to more than one flag in play. At minimum I think we should always be specific here. Our [live table demo page](https://live.d2l.dev/BrightspaceUI/core/components/table/demo/table.html) is actually showing a weird mix of opener fixed positioning + popover.